assets/js/modules/analytics/datastore/__fixtures__/create-account.json

```diff
@@ -0,0 +1,6 @@
+{
+  "accountName": "Test Account",
+  "propertyName": "Test Property",
+  "profileName": "Test Profile",
+  "timezone": "Test Timezone."
+}
```

Comment:
One thing that we need to change right away is add a test account ticket ID in here - we'll need to test that such an ID is taken from the API response as expected.